presto-hive/src/main/java/com/facebook/presto/hive/HivePageSourceProvider.java

```diff
@@ -99,6 +99,7 @@
     private final TypeManager typeManager;
     private final RowExpressionService rowExpressionService;
     private final LoadingCache<RowExpressionCacheKey, RowExpression> optimizedRowExpressionCache;
+    private final ColumnConverterProvider columnConverterProvider;
```

Comment:
`HivePageSourceProvider` is related to reader. So it is unlikely we will need the provider in this class. Check other my other comment.

Reply:
Noon question, what is the "reader" in reference to?

Reply:
`PageSource` is the data source of a connector. Or in this case, the data reader.